### ISSUE TITLE
Transfer Currency silently fails

### ISSUE THREAD

**Issue by angrren** [tukui-org/ElvUI#1294]
### Troubleshooting Steps

Nothing happens, it's a bit tricky to debug or trouble shoot. The LUA error suggests that transferring currency is a protected function though, unsure how you'd address this. Disabling ElvUI allows me to transfer currency however.

### Describe the expected behavior and what actually happened?

Currency does not transfer

### Reproducing the issue.

1) Go to Currency Panel, click a currency that can be Warband Transferred, e.g. Residual Memories.
2) On the Currency Options, Click the Transfer button
3) Transfer some currency over to the current character from another by putting a value in the "Amount to Transfer" box, click Confirm button
4) You get the "clunk" sound in the background, but nothing transfers

### Lua Errors

```shell
1x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'RequestCurrencyFromAccountCharacter()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `RequestCurrencyFromAccountCharacter'
[string "@Blizzard_TokenUI/Blizzard_CurrencyTransfer.lua"]:331: in function <...ddOns/Blizzard_TokenUI/Blizzard_CurrencyTransfer.lua:328>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
}
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by kodewdle** [comment]
can you provide a screenshot of `/estatus` after `/edebug on` including this error beside it?  also, please explain how you access currency panel, with keybind or?

**Comment by angrren** [comment]
/estatus after /edebug on:

![image](https://github.com/user-attachments/assets/97e3bcbe-e5b7-47a3-ae32-2922c244cab1)

Unfortunately I cannot capture the LUA error now as all other addons were disabled with edebug on so I don't have access to BugSack. It is now a totally silent failure.

I go via Character Pane keybind -> click Currency tab. I get the same result if I go via middle mouse button on the minimap -> Character Info -> click Currency tab.

**Comment by eivl** [comment]
I have to disable ELVUI to transfer this currency. Is there a need to share more details on the subject?

**Comment by Griznah** [comment]
This works for me today

**Comment by kodewdle** [comment]
thank you, i believe we resolved this bug in 13.75 by this commit: https://github.com/tukui-org/ElvUI/commit/1998a91da766243b3e8ab0fcc8d569ac4b66b549

**Comment by tawilliams** [comment]
Last comment says closed completed, but this issue still persists as of this comment. Does have a "possibly unfixable" label.
